backend/typescript/graphql/resolvers/entityResolvers.ts

```diff
@@ -11,30 +16,59 @@ const entityResolvers = {
     entities: async () => {
       return entityService.getEntities();
     },
+    file: async (_req: any, { fileUUID }: { fileUUID: string }) => {
+      return fileStorageService.getFile(fileUUID);
+    },
   },
   Mutation: {
     createEntity: async (
       _req: any,
-      { entity }: { entity: EntityRequestDTO },
+      { entity, file }: { entity: EntityRequestDTO; file: Promise<FileUpload> },
     ) => {
+      let filePath = "";
+      let fileContentType = "";
+      if (file) {
+        const { createReadStream, mimetype, filename } = await file;
+        const uploadDir = "uploads";
+        filePath = `${uploadDir}/${filename}`;
+        fileContentType = mimetype;
+        createReadStream().pipe(fs.createWriteStream(filePath));
```

Comment:
Should probably await the completion of the write like in the docs: https://www.apollographql.com/docs/apollo-server/data/file-uploads/

```ts
import { finished } = from "stream/promises";

const out = fs.createWriteStream(filePath);
createReadStream().pipe(out);
await finished(out);
```